code/bmpman/bmpman.cpp

```diff
@@ -2854,6 +2854,38 @@ int bm_release(int handle, int clear_render_targets) {
 	return 1;
 }
 
+bool bm_release_rendertarget(int handle) {
+	Assert(handle >= 0);
+
+	bitmap_entry* be = bm_get_entry(handle);
+
+	if (be->type == BM_TYPE_NONE) {
+		return false;	// Already been released?
+	}
+
+	Assertion(be->handle == handle, "Invalid bitmap handle number %d (expected %d) for %s passed to bm_release_rendertarget()\n", be->handle, handle, be->filename);
+
+	if (!((be->type == BM_TYPE_RENDER_TARGET_STATIC) || (be->type == BM_TYPE_RENDER_TARGET_DYNAMIC))) {
+		nprintf(("BmpMan", "Tried to release a render target of a non-rendered bitmap!\n"));
+		return false;
+	}
+
+	if (bm_is_anim(be)) {
```

Comment:
Render targets are never animations so this is not needed. You could even turn it into an assertion so that we catch this when we get into some weird state.

Reply:
Okay. I wasn't sure about that, but thanks for confirming.
Assert or return 0; with debug message?

Reply:
Assert is appropriate here since a failure of that condition indicates a programming error.